README.md

```diff
@@ -7,7 +7,6 @@ If you have any questions, or simply want to say hi, please give us a shout at h
 
 > We have received a number of generous offers to contribute front-end developer effort to nextstrain (and auspice) folowing our work on [SARS-CoV-2](https://nextstrain.org/ncov).
 We would be grateful for code contributions, as well as constructive criticism and advice.
-A list of potential issues is being actively maintained at https://github.com/orgs/nextstrain/projects/5.
```

Comment:
Good to clean this up, but frankly, I'd say remove this whole paragraph and add an additional sentence to the preceding one, something like: "We welcome contributions to Nextstrain; see [our contributing documentation](https://docs.nextstrain.org/en/latest/guides/contribute/) to learn more."

Reply:
Done: 1398a282223c08bb006725d37a9a9ff0392ba093